Update index.md

gveneyjr opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

Fixing a very small syntax error for the benefit of newer coders.

<!-- ✍️ Summarize your changes in one or two sentences -->

### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->


<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

Commit:
Update index.md

Josh-Cena commented on files/en-us/web/api/service_worker_api/using_service_workers/index.md:
```diff
@@ -153,9 +153,9 @@ Now you've got your site assets cached, you need to tell service workers to do s
    self.addEventListener("fetch", (event) => {
      event
-       .respondWith
-       // custom content goes here
-       ();
+       .respondWith(
+          // custom content goes here
+       );
    });
```
This is formatted by Prettier, so we need something that Prettier is happy with.

```js
   self.addEventListener("fetch", (event) => {
     event.respondWith(/* custom content goes here */);
   });
```

gveneyjr replied:
Hello! I'm new to open source so not sure if I'm following the intended procedure. I submitted another commit with your suggestion for Prettier compatibility but my main objective was just to let you all know. If it is simpler for you all to fix it without the pull request process that is perfectly fine too! Cheers.

Commit:
Update index.md

Changing code to be in-line with reviewer's request and idea for Prettier compatibility.

Josh-Cena committed:
Update index.md

Josh-Cena reviewed (approved):
Thanks, and welcome to MDN :)
